micromasters_import/management/commands/queries/002_courserun_enrollment.sql

```diff
@@ -0,0 +1,83 @@
+-- load postgres_fdw extension on MITxOnline DB
+CREATE EXTENSION IF NOT EXISTS postgres_fdw;
+
+--- create a foreigner server to connect to external MicroMaster DB
+CREATE SERVER IF NOT EXISTS micromaster_bridge
+	FOREIGN DATA WRAPPER postgres_fdw
+    OPTIONS (host '--------', dbname '--------');
+
+-- setup a user mapping to authenticate:
+CREATE USER MAPPING IF NOT EXISTS FOR postgres
+	SERVER micromaster_bridge
+    OPTIONS (user '-------', password '------');
```

Comment:
I think you can remove these queries - the db user won't have permission to do these kinds of operations and it's going to be something devops sets up ahead of time anyway. Locally it'll need to be run manually or as part of an init script.

Reply:
oh ok. Good to know. I will remove that